tracer/build/artifacts/after-install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+ln -sf /opt/datadog/dd-dotnet.sh /usr/bin/dd-dotnet
```

Comment:
Is it possible to install to non-standard locations? If so, can we handle that in these scripts?

Reply:
After a quick search, it looks like it's _technically_ possible to install to another location (by manually extracting the content of the package) but it's not supported.